tests/test_time.py

```diff
@@ -0,0 +1,159 @@
+        Test cases are generated from the Global Monitoring Laboratory and the
+        Government of Australia.[1]_[2]_
+
+        References
+        ----------
+        .. [1] NOAA US Department of Commerce. ESRL Global Monitoring
+           Laboratory -Global Radiation and Aerosols. url:
+           https://gml.noaa.gov/grad/solcalc/.
+        .. [2] Time Conventions. url:
+           http://www.bom.gov.au/climate/data-services/solar/content/data-time.html.
+        """
+
+        julian = [2455368.75, 2459450.85, 2456293.5416666665]
+        longitude = [-105, -118.24, 147.46]
+
+        expected_true_solar_time = np.array([23.0715, 0.378059, 10.76556])
+        actual_true_solar_time = Time(julian).true_solar_time(longitude).to(u.hourangle)
+
+        self.assertTrue(np.allclose(expected_true_solar_time,
+                                    actual_true_solar_time, atol=0.11))
+
+    def test_mean_solar_time_for_validation(self):
+        """Test `Time.mean_solar_time`.
+
+        Notes
+        -----
+        Test cases are generated from the Government of Australia.[1]_
+
+        References
+        ----------
+        .. [1] Time Conventions. url:
+           http://www.bom.gov.au/climate/data-services/solar/content/data-time.html.
+        """
+
+        julian = 2456293.5416666665
+        longitude = 147.46
+
+        expected_mean_solar_time = np.array([10.83056])
+        actual_mean_solar_time = Time(julian).mean_solar_time(longitude).to(u.hourangle)
+
+        self.assertTrue(np.allclose(expected_mean_solar_time,
+                                    actual_mean_solar_time, atol=0.001))
+
+    def test_true_hour_angle_for_validation(self):
+        """Test `Time.true_hour_angle`.
+
+        Notes
+        -----
+        Test cases are taken from Global Monitoring Laboratory.[1]_
+
+        References
+        ----------
+        .. [1] NOAA US Department of Commerce. ESRL Global Monitoring
+           Laboratory -Global Radiation and Aerosols. url:
+           https://gml.noaa.gov/grad/solcalc/.
+        """
+
+        julian = [2455368.75, 2459450.85, 2456293.5416666665]
+        longitude = [-105, -118.24, 147.46]
+
+        expected_true_hour_angle = np.array([166.0734, 185.671, 341.53]) / 15 % 24
+        actual_true_hour_angle = Time(julian).true_hour_angle(longitude).to(u.hourangle)
+
+        self.assertTrue(np.allclose(expected_true_hour_angle,
+                                    actual_true_hour_angle, atol=1.51))
+
+    def test_mean_hour_angle_for_validation(self):
+        julian = 2456293.5416666665
+        longitude = 147.46
+
+        true_mean_hour_angle = np.array([342.4584]) / 15 % 24
+        test_mean_hour_angle = Time(julian).mean_hour_angle(longitude).to(u.hourangle)
+
+        self.assertTrue(np.allclose(true_mean_hour_angle, test_mean_hour_angle,
+                        atol=0.01))
+
+    def test_ut1_for_validation(self):
+        ut1_utc_diff = self.astropy_time.get_delta_ut1_utc().value / 3600
```

Comment:
You can probably convert this value into a certain unit and get the value rather than using constants. Look into this and apply to all similar instances.